tests/e2e/standard/test_init.py

```diff
@@ -92,6 +94,36 @@ def test_git_disabled(self, environment):
         exp.sync()
         assert not exp.exists("source_code/git")
 
+    def test_infer_dependencies(self, environment):
+        exp = neptune.init_run(
```

Comment:
This will produce a atexit error as run wasn't stopped. Use contextmanager syntax here (`with neptune.init_run(....) as exp:`

Reply:
Shall I correct it in other tests that were already here, that don't implement it like that? Like `test_git_disabled`

Reply:
I would be grateful.